src/app/admin/admin-reports/filtered-items/filtered-items.component.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { waitForAsync, ComponentFixture, TestBed} from '@angular/core/testing';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { TranslateLoader, TranslateModule, TranslateService } from '@ngx-translate/core';
```

Comment:
This line has lint errors because you don't use `TranslateService` in this spec. It can be removed from the import.